app/src/main/java/com/github/polypoly/app/map/MapActivity.kt

```diff
@@ -69,7 +87,9 @@ class MapActivity : ComponentActivity() {
     private val initialZoom = 18.0
     private val markerSideLength = 100
 
-    private val markerToLocalization = mutableMapOf<Marker, Localization>()
+    private val maxCloseLocationDistance = 10.0
```

Comment:
I do not quite get what that variable is used for just with its name --> could you maybe come up with a better one?

Reply:
a comment should do the trick, or else the name would be way too long even for a static constant